### PR TITLE
ENH: Add i/o prefix to input and output parameters

### PR DESCRIPTION
**Pull request by jakereps**

### CODE REVIEW

File: q2cli/cli.py [hunk 1]
```diff
@@ -237,6 +239,15 @@ def _build_visualizer_command(name, visualizer):
                          help=visualizer.description)
 
 
+def _filter_kwargs(kwargs):
```
Comment (gregcaporaso) [inline q2cli/cli.py]: The names used here are a little confusing. What about these changes:

* ``_filter_kwargs`` to ``_option_names_to_api_names`` (filtering kwargs would suggest to me that you're getting rid of some of them) 
* ``stripped_kwarg`` to ``api_name``
* ``stripped_kwargs`` (line 127) to ``option_name_map``

**Comment by gregcaporaso** [comment]
Just one comment on this @jakereps.

**Comment by jakereps** [comment]
Thanks @gregcaporaso, should be good to go!

**Comment by ebolyen** [comment]
:tada:

**Pull request merged**